### ISSUE TITLE
QAT using pytorch-quantization cause accuracy loss after exporting to onnx

### ISSUE THREAD

**Issue by MomoP32** [pytorch/pytorch#120166]
### 🐛 Describe the bug

I use pytorch-quantization to do QAT for a pointpillar model, it works fine during pytorch training, however, when I export the torch model to onnx, accuracy degrades badly. In my opinion, this process should have no loss of accuracy at all.

I report this issue against TRT repo, the developer assigned thinks that should be a bug of pytorch.
https://github.com/NVIDIA/TensorRT/issues/3654

### Versions

TensorRT Version:
8.5.2
NVIDIA GPU:
RTX3060
NVIDIA Driver Version:
470.161.03
CUDA Version:
11.3
CUDNN Version:
8.6

Operating System:
Ubuntu 20.04
Python Version (if applicable):
3.8
PyTorch Version (if applicable):
1..11
pytorch-quantization:
2.1.2

**Comment by MomoP32** [comment]
I give up the method of using explicit quantization, but introduced fake-quantization operators during training, and use the PTQ method to quant, and the result improves significantly compared to w/o QAT.  So, I think something wrong about fake-quantization node in onnx.(Q/DQ)

**Comment by lucy3589** [comment]
I have the same problem you have. The accuracy of onnx model is lower than that of pt.

**Comment by supriyar** [comment]
What is the accuracy of the model before exporting to onnx? If that looks good, then the issue seems to be with the onnx operator - perhaps checking with https://github.com/microsoft/onnxruntime/issues/ would be more productive for you.

**Comment by lucy3589** [comment]
> What is the accuracy of the model before exporting to onnx? If that looks good, then the issue seems to be with the onnx operator - perhaps checking with https://github.com/microsoft/onnxruntime/issues/ would be more productive for you.

@supriyar hi, pt_map50-95(box)0.35 pt_map50-95(mask)0.29 ;    onnx_map50-95(box)0.29   onnx_map50-95(mask)0.25

**Comment by MomoP32** [comment]
> What is the accuracy of the model before exporting to onnx? If that looks good, then the issue seems to be with the onnx operator - perhaps checking with https://github.com/microsoft/onnxruntime/issues/ would be more productive for you.

The model's AP dropped by less than 2, which is fine I think.  I will check in onnxruntime's issues, thanks.

**Comment by MomoP32** [comment]
This problem is solved in pytorch_quantization repo.
https://github.com/NVIDIA/TensorRT/issues/3654

**Comment by HaoLiuHust** [comment]
> I give up the method of using explicit quantization, but introduce fake-quantization operators during training, and use the PTQ method to quant, and the result improves significantly compared to w/o QAT. So, I think something wrong about fake-quantization node in onnx.(Q/DQ)

how to do this,  PTQ is done by pytorch or onnx?
